src/devices/controller.c

```diff
@@ -259,7 +259,7 @@ int32_t controller_print(controller_id_e_t id, uint8_t line, uint8_t col, const
 }
 
 int32_t controller_clear_line(controller_id_e_t id, uint8_t line) {
-	static const char* clear = "               ";
+	static const char* clear = "                   ";
```

Comment:
If you want to be fancy, you could assert that the lengths match up.

Reply:
like, making it so that it prints " " * Max string length?

Reply:
Does `char*` clear = ' ' * CONTROLLER_MAX_COLS` not work in C?

Reply:
I don't think so... you'd probably have to use a for loop for it.

Reply:
strings aren't first class citizens in C. While what you have will do something, it won't do what you expect unless you want to know what 32 * 19 is.

Reply:
```c
kassert(strlen(clear) == CONTROLLER_MAX_COLS)
```
Is what I had in mind